build/assets/custom-dictionary.txt

```diff
@@ -128,3 +130,5 @@ underperform
 mRNA
 generalizable
 counterintuitively
+genotypic
+mahadeva
```

Comment:
I only see this in a citation...shouldn't manubot not spellcheck citations? (i.e. is this a bug?)